src/css/brandColors.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable n/no-sync */
+/* eslint-disable jsdoc/require-jsdoc */
+import path from 'path';
+
+import { getCssColorVariables } from '../../utils';
+import jsonData from '../figma/brandColors.json';
+
+describe('CSS Variables', () => {
+  const cssFilePath = path.join(__dirname, 'generated-brand-colors.css');
+  const cssVariables = getCssColorVariables(cssFilePath, 'brand-colors');
+
+  it('should match JSON values', () => {
+    for (const [colorGroup, shades] of Object.entries(jsonData)) {
+      for (const [shade, details] of Object.entries(shades)) {
+        const variableName = `${colorGroup}-${shade}`;
+        expect(cssVariables[variableName]).toBe(details.value);
+      }
+    }
+  });
```

Comment:
Nice work added a console log to see what's happening here. LGTM!

![Screenshot 2024-05-29 at 12 09 34 PM](https://github.com/MetaMask/design-tokens/assets/8112138/0ccd5d19-c111-4320-a248-e2191792959a)